catroid/src/main/java/org/catrobat/catroid/common/Constants.java

```diff
@@ -99,8 +99,11 @@
 
 	// Web:
 	private static final String MAIN_URL_PRODUCTION = "https://share.catrob.at";
-	private static final String MAIN_URL_WEBTEST = "https://web-test.catrob.at";
-	public static final String MAIN_URL_HTTPS = BuildConfig.USE_WEB_TEST ? MAIN_URL_WEBTEST : MAIN_URL_PRODUCTION;
+	private static final String WEB_TEST_URL = BuildConfig.WEB_TEST_URL;
+	public static final String MAIN_URL_HTTPS = BuildConfig.WEB_TEST_FLAG ? WEB_TEST_URL : MAIN_URL_PRODUCTION;
+
```

Comment:
Too many empty lines should be removed see Static analyisis warning https://jenkins.catrob.at/job/Catroid/job/PR-3433/2/analysis/fileName.1858406006/source.ba519b23-505d-4a34-aaeb-045a9126df40/#103